pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -622,15 +622,15 @@ var _ = Describe("Converter", func() {
   <iothreads>3</iothreads>
 </domain>`, domainType, "%s")
 		var convertedDomainWith5Period = fmt.Sprintf(convertedDomain,
-			`<memballoon model="virtio">
+			`<memballoon model="virtio-non-transitional">
```

Comment:
Threr exists a unit test her which uses a function called `ExpectVirtioTransitionalOnly`, would you mind adjusting the checks in this function to also check the ballooning device?

Reply:
Never mind, you did that.